go.mod

```diff
@@ -127,7 +127,7 @@ require (
 	golang.org/x/exp v0.0.0-20240103183307-be819d1f06fc
 	golang.org/x/tools v0.24.0
 	gopkg.in/yaml.v3 v3.0.1
-	honnef.co/go/tools v0.4.7
+	honnef.co/go/tools v0.5.0-rc.1
```

Comment:
Note this is a pre-release of static check.

Reply:
staticcheck just release v0.5.0

Reply:
Should you update to v0.5.0 for this release?

Reply:
nvm 👍

Reply:
yes it's a requirement to support go1.23